endpoints-control/src/main/java/com/google/api/control/model/KnownMetrics.java

```diff
@@ -110,12 +86,18 @@
   private MetricKind kind;
   private ValueType type;
   private Update updater;
+  private Mark mark;
 
   private KnownMetrics(String name, MetricKind kind, ValueType type, Update updater) {
+    this(name, kind, type, updater, Mark.PRODUCER);
```

Comment:
Feels a little strange to see the default mark is producer.

I guess it doesn't matter too much since this is private